Packs/Tenable_io/Integrations/Tenable_io/Tenable_io.py

```diff
@@ -1,15 +1,15 @@
-import demistomock as demisto  # noqa: F401
-from CommonServerPython import *  # noqa: F401
+register_module_line('Tenable.io', 'start', __line__())
+### pack version: 2.1.13
+
+
```

Comment:
Please remove these lines. They're added automatically for integrations on the server, but not relevant here in the source code.
```suggestion
```